src/components/LoginButton.jsx

```diff
@@ -2,8 +2,8 @@ import React from 'react';
 import auth from 'solid-auth-client';
 
 /** Button that lets the user log in with Solid. */
-export default function LoginButton({ popup }) {
+export default function LoginButton({ popup, className }) {
   return <button
-    className="solid auth login"
+    className={`solid auth login ${className}`}
```

Comment:
Maybe we should directly have

```
export default function LoginButton({ popup, className="solid auth login" }) {
  return <button
    className={className}
```

so people can also get rid of the built-in classes?

Reply:
I am fine with that.